supplier_product/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+from supplier_product.models import SupplierProduct, Supplier
+
+
+class SupplierProductAdmin(admin.ModelAdmin):
+    list_display = ('suppler_product_id', 'qr_code', 'user_name', 'price', 'quantity')
+
+
+class SupplierAdmin(admin.ModelAdmin):
+    list_display = ('user_name', 'first_name', 'last_name',
+                    'password', 'business_name',)
+
+
+admin.site.register(Supplier, SupplierAdmin)
```

Comment:
Since supplier is merged ,delete these lines because it is a duplicate code from supplier that already been done by paz